# STAT 207 Lab 7- Group Part - [5 total points] - Solutions

## More Confidence Intervals

Due: Tuesday, October 11, 11:59pm CST

## <u>Case Study</u>: Former STAT107 Student Analysis

### Motivation
Starting this Fall, STAT107 and STAT207 (as well as other courses) now count as requirements towards a brand new suite of majors that we are calling "X+Data Science" majors. Thus, we in the statistics department are interested to learn more about the type of students that enroll in courses like STAT107. In the previous four semesters, 497 students enrolled in STAT107. Rather than track down and ask each of these 497 students questions, we will use the students in *this lab section* (who have presumably taken STAT107) in an attempt to learn more about the larger population of **all students who have taken STAT107**.

<hr>

<p>&nbsp;</p>
<table style="border: none;border-collapse: collapse;width:102pt;">
    <tbody>
        <tr>
            <td style="color:black;font-size:15px;font-weight:700;font-style:normal;text-decoration:none;font-family:Calibri, sans-serif;text-align:right;vertical-align:bottom;border:.5pt solid windowtext;height:14.25pt;width:51pt;">Problem</td>
            <td style="color:black;font-size:15px;font-weight:700;font-style:normal;text-decoration:none;font-family:Calibri, sans-serif;text-align:general;vertical-align:bottom;border:.5pt solid windowtext;border-left:none;width:51pt;">Points</td>
        </tr>
        <tr>
            <td style="color:black;font-size:15px;font-weight:400;font-style:normal;text-decoration:none;font-family:Calibri, sans-serif;text-align:right;vertical-align:bottom;border:.5pt solid windowtext;height:14.25pt;border-top:none;">1.1</td>
            <td style="color:black;font-size:15px;font-weight:400;font-style:normal;text-decoration:none;font-family:Calibri, sans-serif;text-align:right;vertical-align:bottom;border:.5pt solid windowtext;border-top:none;border-left:none;">0.5</td>
        </tr>
        <tr>
            <td style="color:black;font-size:15px;font-weight:400;font-style:normal;text-decoration:none;font-family:Calibri, sans-serif;text-align:right;vertical-align:bottom;border:.5pt solid windowtext;height:14.25pt;border-top:none;">1.2</td>
            <td style="color:black;font-size:15px;font-weight:400;font-style:normal;text-decoration:none;font-family:Calibri, sans-serif;text-align:right;vertical-align:bottom;border:.5pt solid windowtext;border-top:none;border-left:none;">1</td>
        </tr>
        <tr>
            <td style="color:black;font-size:15px;font-weight:400;font-style:normal;text-decoration:none;font-family:Calibri, sans-serif;text-align:right;vertical-align:bottom;border:.5pt solid windowtext;height:14.25pt;border-top:none;">2</td>
            <td style="color:black;font-size:15px;font-weight:400;font-style:normal;text-decoration:none;font-family:Calibri, sans-serif;text-align:right;vertical-align:bottom;border:.5pt solid windowtext;border-top:none;border-left:none;">1</td>
        </tr>
        <tr>
            <td style="color:black;font-size:15px;font-weight:400;font-style:normal;text-decoration:none;font-family:Calibri, sans-serif;text-align:right;vertical-align:bottom;border:.5pt solid windowtext;height:14.25pt;border-top:none;">3</td>
            <td style="color:black;font-size:15px;font-weight:400;font-style:normal;text-decoration:none;font-family:Calibri, sans-serif;text-align:right;vertical-align:bottom;border:.5pt solid windowtext;border-top:none;border-left:none;">1</td>
        </tr>
        <tr>
            <td style="color:black;font-size:15px;font-weight:400;font-style:normal;text-decoration:none;font-family:Calibri, sans-serif;text-align:right;vertical-align:bottom;border:.5pt solid windowtext;height:14.25pt;border-top:none;">4</td>
            <td style="color:black;font-size:15px;font-weight:400;font-style:normal;text-decoration:none;font-family:Calibri, sans-serif;text-align:right;vertical-align:bottom;border:.5pt solid windowtext;border-top:none;border-left:none;">1</td>
        </tr>
        <tr>
            <td style="color:black;font-size:15px;font-weight:400;font-style:normal;text-decoration:none;font-family:Calibri, sans-serif;text-align:right;vertical-align:bottom;border:.5pt solid windowtext;height:14.25pt;border-top:none;">5</td>
            <td style="color:black;font-size:15px;font-weight:400;font-style:normal;text-decoration:none;font-family:Calibri, sans-serif;text-align:right;vertical-align:bottom;border:.5pt solid windowtext;border-top:none;border-left:none;">0.5</td>
        </tr>
    </tbody>
</table>



In [7]:
import math

## <u>Part 1</u>:  Create and Conduct a Survey

### 1.1. Create a Research Question

Create a research question that *you* would be interested in asking all former STAT107 students.
Your question should have a **categorical response with just two answers**, for instance: 

<u>Ex:</u> ```"Have you ever run through a corn field before?: Yes/No"```

**<u>Notes</u>:**
* Don't use this example question. You should come up with your own.
* If you were not able to attend the lab, then you can pretend that you asked the students in your lab this question:
```"Have you ever run through a corn field before?"```

Write your question below.

Are you from illinois? 

### 1.2. Collect a Random Sample

Then, ask your question to n=6 randomly selected students in this *lab section and record your answers*. Put these six responses (not the names) in a dataframe.

*To ensure that your sample of students in this lab is random, see your TA or the course assistants for your random sample of 6 students in this lab.*

**<u>Notes</u>:**
* You are not **required** to answer any survey questions that you do not feel comfortable answering that someone else asks you.
* If you were not able to attend the lab, then here is a random sample of six students from the class and their answers to your question: ```['yes', 'yes', 'no', 'no', 'no', 'no']```.

<hr>

In [1]:
survey = ['yes', 'no','no','no','no','no']
survey

['yes', 'no', 'no', 'no', 'no', 'no']

<hr>

## <u> Part 2</u>:  Calculate Sample Proportion

Then, using your sample that you just collected, calculate the proportion that are of a particular level that you are interested in.

In [6]:
1/6

0.16666666666666666

<hr>

## <u>Part 3</u>: Checking Confidence Interval Conditions

In part 4, we would like to use your sample to create a plausible range of values for a **population proportion** corresponding to our population of **all former STAT107 students** and the question that you asked. Specifically, we will create a **99% confidence interval** for this **population proportion**.

Check the appropriate corresponding Central Limit Theorem conditions needed in order to create and use this confidence inteval accurately.

<hr>

* 6 < 10% of the population of STAT 107 students 
* np = 6(0.1667) = 1.0002 < 10 and n(1-p) = 4.9998 < 10
* The central limit theorem doesnt work becuase second condition isnt met so proceed with caution 

## <u>Part 4</u>: Creating the Population Proportion Confidence Interval

Use your sample to create a **99% confidence interval** for the corresponding **population proportion** value.

**Note:**
* If you found that not all of your conditions from part 3 were met, then you should "proceed with caution". This means that you should still create this confidence interval (for the purpose of this assignment), but knowing that the conclusions that we are drawing from this confidence interval may not be completely accurate



In [13]:
std_error = math.sqrt((0.1667*(1-0.1667))/6)
std_error

0.15215732537957766

In [14]:
upper_bound = 0.1667 + 2.576*(std_error)
upper_bound

0.558657270177792

In [16]:
lower_bound = 0.1667 - 2.576*(std_error)
lower_bound

-0.22525727017779204

## <u>Part 5</u>: Confidence Interval Interpretation

Finally, put your confidence interval into words.

We are 99 percent confident that the population proportion of former STAT 107 students that are from illinois is between -0.225 and 0.558

### <u>Contribution Report</u>:

All worked on everything together